### DISCUSSION TITLE
Specify an IP address for SMTP endpoint

### DISCUSSION THREAD

**Discussion by TheBlueKingLP** [postalserver/postal#2558] [Help with using Postal]
How can I specify an alternative IP address for reaching the SMTP endpoint?

My SMTP endpoint is in a docker container and the DNS record is pointing to my public IP address, which won't help it locating the SMTP endpoint using the correct docker network.

I tried to put the container name of the SMTP endpoint but it says `SSL_connect returned=1 errno=0 peeraddr=172.24.0.17:25 state=error: certificate verify failed (Hostname mismatch)` which is normal because obviously the docker container name didn't match the hostname.

I also tried to add `extra_hosts` to the docker compose file using a `docker-compose.override.yml` file but seems like it isn't being picked up by the postalserver 

Edit: to clarify, the SMTP endpoint is on the same machine as the postalserver.io containers. I added them to the same docker bridge network.

**Comment by willpower232** [comment]
I imagine you would have to give the Postal config file the docker host IP address so that it could connect the dots but I'm not sure.

This would be why it is recommended to run with host networking as per the official installer.

**Reply by TheBlueKingLP** [reply]
unfortunately that is not possible, as those ports are already being used by another docker containers, that is why I need macvlan to assign the containers another IP address
